db/migrate/20230731090232_add_fields_to_accounts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddFieldsToAccounts < ActiveRecord::Migration[6.1]
+  def change
+    change_table :accounts, bulk: true do |t|
+      t.string :domain
+      t.string :organization
+      t.jsonb  :services, default: []
+      t.references :account_manager, foreign_key: { to_table: :users }
+      t.boolean :owner_request, null: false, default: false
+      t.text :owner_request_notes
+      t.boolean :mfa_supported, null: false, default: false
```

Comment:
Itt mintha eltértél volna az issue-tól.
- MFA needed: szerintem nem volt benne a listában
- 2 db "Notes" és "Extra Notes"-ból az egyik mintha hiányozna

Kérlek beszélj a megrendelővel, hogy mire van szüksége, majd ezt vezessétek át az issue alá is.